File: test/static/code/java/GenericExceptionsClose.java
import java.security.SecureRandom;

class GenericExceptionsClose{
   public static void main(String args[]){
     try{
         int a[]=new int[7];
         SecureRandom random = new SecureRandom();
         byte bytes[] = new byte[20];
         a[0]=random.nextBytes(bytes);
         a[4]=30/0;
         System.out.println("First print statement in try block");
     }
     catch(ArithmeticException e){
        System.out.println("Warning: ArithmeticException");
     }
     catch(ArrayIndexOutOfBoundsException e){
        System.out.println("Warning: ArrayIndexOutOfBoundsException");
     }
/*
     try {
           System.out.println("Out of try-catch block...");
           int a = Math.random();
  catch(Exception e){
        System.out.println("Warning: Some Other exception");
     }
*/
  }
}

File: test/static/code/csharp/GenericExceptionsClose.cs
using System;

public class GenericExceptionsClose
{
  public static void Main()
  {
    try
    {
      int[] a = new int[7];
      a[4]=30/0;
      Console.WriteLine("First print statement in try block");
      if (a > 5) {
        Console.WriteLine("Big num")
      } else {
        Console.WriteLine("Small num")
      }
    }

    catch(ArithmeticException e)
    {
      Console.WriteLine("Warning: ArithmeticException");
    }

    catch(IndexOutOfRangeException e)
    {
      Console.WriteLine("Warning: ArrayIndexOutOfBoundsException");
    }
/*
    try
    {
      Console.WriteLine("Out of try-catch block...");
      if (a > 5) {
        Console.WriteLine("Big num")
      }
    }

    catch(Exception e)
    {
      Console.WriteLine("Warning: Some Other exception");
    }
*/
  }
}
